test/helpers/pages/queryservice-ui/queryservice-ui.page.ts

```diff
@@ -14,7 +14,6 @@ class QueryServiceUIPage extends SubmittablePage {
 	 *
 	 * @param {string} query
 	 * @param {string[]} prefixes - Optional
-	 * @return {void}
```

Comment:
This introduced a lint warning - please fix

Reply:
Weird, I think it was removed by some linter run. I definitely did not remove it manually. Thanks for the catch. Readded.